### DISCUSSION TITLE
支持PCL原版与快照版同时开启

### DISCUSSION THREAD

**Discussion by kong-que-dnf123** [Meloong-Git/PCL#1444] [提问] [answered]
PCL原版和快照版不能同时开启，想开一个必须先关另一个，不然会回到已经开的PCL
（我想上快照版，可许多资源目前都在原版上啊）

**Comment by Chiloven945** [comment]
你可以把原版的exe换成快照的啊...... 只用这样子就行了啊，除非你两个都想拥有可以去交个issue

**Comment by WForst-Breeze** [comment, marked as answer]
PCL2 目前不支持多开，也不会支持多开，因为这会导致一些相关的、不必要的问题出现
